test/integration/api-support/test/index.test.js

```diff
@@ -44,6 +44,17 @@ function runTests(dev = false) {
     expect(res.status).not.toEqual(404)
   })
 
+  it('should set cors headers when adding cors middleware', async () => {
+    const res = await fetchViaHTTP(appPort, '/api/cors', null, {
+      method: 'OPTIONS',
+    })
```

Comment:
```suggestion
      headers: {
        origin: 'example.com'
      }
    })
```
Should we set the `origin` header to ensure this isn't overridden when requesting from the client? We opt-out of overriding the CORS headers if there is no `origin` header so this test wouldn't fail previously since it's using `node-fetch` and it doesn't add the `origin` header by default

Reply:
Great point!